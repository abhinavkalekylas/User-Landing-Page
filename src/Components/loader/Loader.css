.loader {
    align-items: center;
    display: flex;
    margin-top: 12px;
}

.loader > h2, div {
    color: white;
}