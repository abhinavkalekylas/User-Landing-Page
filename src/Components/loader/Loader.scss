.loader_container {
    align-items: center;
    display: flex;
    margin-top: 2rem;

    .load_text, .spinner {
        color: white;
    }
}