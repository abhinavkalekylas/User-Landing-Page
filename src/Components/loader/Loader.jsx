import React from 'react'
import "./Loader.css"

const Loader = () => {
  return (
    <div className="container loader">
        <h2>Loading...  Please wait..</h2>
        <div className="spinner-border ms-auto" role="status" aria-hidden="true"></div>
    </div>
  )
}

export default Loader