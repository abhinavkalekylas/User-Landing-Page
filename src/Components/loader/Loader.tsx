import React from "react";
import "./Loader.scss";

const Loader = () => {
  return (
    <div className="container loader_container">
      <h2 className="load_text">Loading... Please wait..</h2>
      <div
        className="spinner spinner-border ms-auto"
        role="status"
        aria-hidden="true"
      ></div>
    </div>
  );
};

export default Loader;
