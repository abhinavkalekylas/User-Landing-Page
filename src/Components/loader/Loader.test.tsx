import toJson from 'enzyme-to-json';
import { shallow } from "enzyme";
import Loader from "./Loader"

// snapshot testing
it('renders correctly App component', () => {
    const wrapper = shallow(<Loader />)
    expect(toJson(wrapper)).toMatchSnapshot();
  });