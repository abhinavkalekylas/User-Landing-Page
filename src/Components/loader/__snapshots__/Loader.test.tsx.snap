// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly App component 1`] = `
<div
  className="container loader"
>
  <h2>
    Loading...  Please wait..
  </h2>
  <div
    aria-hidden="true"
    className="spinner-border ms-auto"
    role="status"
  />
</div>
`;
